packages/vaex-astro/vaex/astro/fits.py

```diff
@@ -19,6 +19,7 @@
 
 class FitsBinTable(DatasetMemoryMapped):
     def __init__(self, filename, write=False, fs_options={}, fs=None):
+	snake_name='fits'
```

Comment:
```suggestion
    snake_name='fits'
    def __init__(self, filename, write=False, fs_options={}, fs=None):
```
The snake name should be at the class level

Reply:
I think it also needs the @vaex.dataset.register decorator.